### PR TITLE
docs - updates to installation 

### PR DESCRIPTION
**Pull request by jeff-bruemmer**
- Clean up to make JAR, Docker, and Cloud routes more salient.
- Mention professional services
- Add links to Enterprise downloads.

### CODE REVIEW

File: docs/installation-and-operation/start.md [hunk 1]
```diff
@@ -8,30 +8,15 @@ The birth, care, and feeding of your Metabase.
 
 ## [Installing Metabase](./installing-metabase.md)
```
Comment (abdhabli) [inline docs/installation-and-operation/start.md:9]: Small comment, if we have added the link down, is it better to remove the link from the title?

File: docs/installation-and-operation/running-the-metabase-jar-file.md [hunk 2]
```diff
@@ -17,16 +17,18 @@ If you have a token for the [Pro or Enterprise editions](https://www.metabase.co
 
 If you have Java installed:
 
-1. [Download the JAR file for Metabase OSS](https://metabase.com/start/oss/jar).
-2. Create a new directory and move the Metabase JAR into it.
-3. Change into your new Metabase directory and run the JAR.
+1. [Download the JAR file for Metabase OSS](https://metabase.com/start/oss/jar). If you're on a [Pro](https://www.metabase.com/product/pro) or [Enterprise](https://www.metabase.com/product/enterprise) plan, download the [JAR for the Enterprise Edition](https://downloads.metabase.com/enterprise/latest/metabase.jar).
+1. Create a new directory and move the Metabase JAR into it.
```
Comment (abdhabli) [inline docs/installation-and-operation/running-the-metabase-jar-file.md:21]: Shouldn't this one be 2 and the other one is 3?

**Review by abdhabli — APPROVED** [review]
just left tiny comments 🙌

**Comment by jeff-bruemmer** [comment]
@metabase-bot backport release-x.48.x

**Pull request merged**